src/reporter.ts

```diff
@@ -91,26 +91,35 @@ export default class SauceReporter extends SummaryFormatter {
 
   logTestCase(testCaseFinished: { testCaseStartedId: string }) {
     const testCaseAttempt = this.eventDataCollector.getTestCaseAttempt(testCaseFinished.testCaseStartedId);
-    const parsed = formatterHelpers.parseTestCaseAttempt({
-      snippetBuilder: this.snippetBuilder,
-      supportCodeLibrary: this.supportCodeLibrary,
-      testCaseAttempt
-    });
+    let parsed: any;
+    try {
+      parsed = formatterHelpers.parseTestCaseAttempt({
+        snippetBuilder: this.snippetBuilder,
+        supportCodeLibrary: this.supportCodeLibrary,
+        testCaseAttempt
+      });
+    } catch (e) {
+      console.error('failed to parse test data: ', e);
+      if (e instanceof TypeError) {
+        console.warn('`paths` field might be set multiple times in your config. Please check sauce config as well as Cucumber.js config file.');
```

Comment:
We're leaving `paths` as required in the saucectl config so we should probably only recommend they check their cucumber config.